src/SayMore/ProjectContext.cs

```diff
@@ -74,6 +80,79 @@ public ProjectContext(string projectSettingsPath, IContainer parentContainer)
 			ProjectWindow = _scope.Resolve<ProjectWindow.Factory>()(projectSettingsPath, views);
 		}
 
+		///-------------------------------------------------------------------------------------------------------
+		/// <summary>
+		/// Set the contributor list to the session file from the metafiles
+		/// </summary>
+		/// <param name="sessionsFolder">Session folder path</param>
+		///-------------------------------------------------------------------------------------------------------
+		public static void SetContributorsListToSession(string sessionsFolder)
```

Comment:
Hmm, not easily unit tested without lots of .xml test data...
If you already have test data that this could be tested with a unit test is appropriate.